juntagrico/entity/jobs.py

```diff
@@ -120,9 +120,11 @@ class Meta:
 
 class JobType(AbstractJobType):
     '''
-    Recuring type of job. do not add field here do it in the parent
+    Recuring type of job. do only add fields here you do not need in a onetime job
     '''
 
+    visible = models.BooleanField(_('Sichtbar'), default=True)
```

Comment:
archived on the other hand implies that is somewhat stored at a different place
the idea is to have a consistent naming so the idea of the visible naming pattern is as follows
visible handles visibility on all places where entites uf such a type are listed
special visibility is handled in a specially named field like visible for depot list and overrides visible.
So as we come across other visible flags which violate this pattern we should adapt them.

Also since types are not visible to members anyway one hase to be very creative to understand this flag this way.